multiplayer/src/hooks/useClickedOutside.ts

```diff
@@ -1,20 +1,22 @@
 import { useEffect, RefObject } from "react";
 
 export function useClickedOutside(
-    ref: RefObject<Element | undefined>,
+    refs: RefObject<Element | null>[],
```

Comment:
The essential piece is this change. We can provide additional refs considered inside the click area.